models/issue.go

```diff
@@ -1892,10 +1905,10 @@ func (issue *Issue) ResolveMentionsByVisibility(ctx DBContext, doer *User, menti
 				}
 				if has {
 					checked = append(checked, team.ID)
-					resolved[team.LowerName] = true
+					resolved[issue.Repo.Owner.LowerName+"/"+team.LowerName] = true
 				}
 			}
-			if len(checked) != 0 {
+			if len(checked) > 0 {
```

Comment:
reason to change this?

Reply:
Hmm, It's not usefull.
```suggestion
			if len(checked) != 0 {
```